src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/EnvelopeProcessor.java

```diff
@@ -147,6 +151,29 @@ public static void assertEnvelopeHasPdfs(InputEnvelope envelope, List<Pdf> pdfs)
         }
     }
 
+    /**
+     * Assert envelope contains at least one scannable item
+     * with valid document type and ocr data for new application.
+     * Throws exception otherwise.
+     *
+     * @param envelope to assert against
+     */
+    public static void assertEnvelopeContainsOcrDataForNewApplication(InputEnvelope envelope) {
+
+        if (envelope.jurisdiction.equalsIgnoreCase("SSCS")
+            && Classification.NEW_APPLICATION.equals(envelope.classification)) {
+
+            boolean isValidScannableItems = envelope.scannableItems
+                .stream()
+                .anyMatch(item -> item.documentType.equals(InputDocumentType.SSCS1) && !MapUtils.isEmpty(item.ocrData));
```

Comment:
should probably check that **all** sscs1 documents have ocr data, not just at least one of them

Reply:
There was a long discussion about this at some point and as far as I remember we couldn't get any more specific than that we definitely require at least one sscs1 document with ocr data.